src/components/Text/MarkedText/MarkedText.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import Text from '../Text';
+import styles from './MarkedText.scss';
+
+const { block, elem } = bem({
+    name: 'MarkedText',
+    classnames: styles
+});
+
+const MarkedText = props => {
```

Comment:
yes, that sounds good. However, I would leave it for a separate PR. We need to collect requirements first before implementing it. I think it is now flexible enough to be extendable. Since for now it is only used by Autosuggest, it will be easy to refactor even if there are some breaking changes, since it is internal only.

Reply:
@eszthoff oke